<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppScreen</class>
 <widget class="QMainWindow" name="AppScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1707</width>
    <height>1170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMenuBar" name="menuBar">
      <widget class="QMenu" name="menuCommands">
       <property name="title">
        <string>Options</string>
       </property>
       <addaction name="hostAction"/>
       <addaction name="connectAction"/>
       <addaction name="disconnectAction"/>
       <addaction name="toggleDarkModeAction"/>
       <addaction name="exitAction"/>
      </widget>
      <addaction name="menuCommands"/>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5" rowspan="2">
       <widget class="QTableWidget" name="usersTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>127</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="messageBox"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="messagesField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="sendMessageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="disconnectAction">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="connectAction">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="toggleDarkModeAction">
   <property name="text">
    <string>Toggle Dark Mode</string>
   </property>
  </action>
  <action name="hostAction">
   <property name="text">
    <string>Host</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
